components/dropdown/dropdown-content-mixin.js

```diff
@@ -239,7 +239,7 @@ export const DropdownContentMixin = superclass => class extends LocalizeCoreElem
 
 		this.noAutoClose = false;
 		this.noAutoFit = false;
-		this.noAutoFocus = false;
+		this.noAutoFocus = true;
```

Comment:
I don't quite understand this change... isn't this going to cause dropdowns which used to automatically handle focus (because this wasn't set and was `true` before) to suddenly stop doing that?